front/src/utils/index.ts

```diff
@@ -10,6 +10,36 @@ export function formatToHumanReadableDate(date: Date | string) {
   }).format(parsedJSDate);
 }
 
-export const getLogoUrlFromDomainName = (domainName?: string): string => {
-  return `https://api.faviconkit.com/${domainName}/144`;
-};
+export function sanitizeURL(link: string | null | undefined) {
+  return link
+    ? link.replace(/(https?:\/\/)|(www\.)/g, '').replace(/\/$/, '')
+    : '';
+}
+
+export function getLogoUrlFromDomainName(
+  domainName?: string,
+): string | undefined {
+  const sanitizedDomain = sanitizeURL(domainName);
+
+  if (!sanitizedDomain) return;
+
+  const prioritizeWWW = ['apple.com'].includes(sanitizedDomain);
```

Comment:
let's only do the sanitzing, we don't need to query the www domain as we are going to use favicon.twenty.com that will always return an icon for www and root domains